hw8/my_exception_class.py

```diff
@@ -0,0 +1,18 @@
+class MyCustomException:
```

Comment:
You implemented here a context manager, but the task was to take your `Custom Exception` and add a log record there.